.eslintrc

```diff
@@ -7,6 +7,7 @@
     },
     "extends": "airbnb",
     "rules": {
+        "max-len": "off",
```

Comment:
I disagree with it, I open JS files with `vim` relatively often, and it's a pain in the neck to read enormous lines.
Idem for `cat`, `grep` and all very useful console tools, even if we have large screens.

I suggest to increase the limit to 100 or 120 max. It's a good way to keep short variables or functions names too.